admin/spec/back-end/service/check-start.service.spec.js

```diff
@@ -206,9 +207,15 @@ describe('check-start.service', () => {
       )
       spyOn(pupilDataService, 'sqlUpdateTokensBatch').and.returnValue(Promise.resolve())
       spyOn(checkStartService, 'prepareCheckQueueMessages').and.returnValue(mockPreparedCheckQueueMessages)
-      spyOn(azureQueueService, 'addMessageAsync')
+      spyOn(azureQueueService, 'addMessageAsync').and.returnValue(Promise.resolve())
```

Comment:
Minor comment: even if you don't return a value here it will still work for the spy.

Reply:
Reverted